docs/sources/review/lint-prose/index.md

```diff
@@ -59,3 +59,73 @@ The task title contains a verb and an object. For example: "Create a dashboard".
 
 <!-- vale Grafana.Quotes = YES -->
 ```
+
+## Use Vale in VS Code
+
+You can use Vale to lint your current document in VS Code.
+
+1. Clone the [Technical Documentation](https://github.com/grafana/technical-documentation) repository.
+
+   ```bash
+   git clone git@github.com:grafana/technical-documentation.git
+   ```
+
+1. Download and install [Vale](https://github.com/errata-ai/vale/releases).
+
+   {{% admonition type="note" %}}
+   Refer to the Linux installation steps at [GitHub Releases](https://vale.sh/docs/vale-cli/installation/#github-releases). Verify that you are downloading the latest build of Vale for Linux.
+   {{% /admonition %}}
+
+   {{< code >}}
+
+   ```linux
+   wget https://github.com/errata-ai/vale/releases/download/v2.28.0/vale_2.28.0_Linux_64-bit.tar.gz
+   mkdir bin && tar -xvzf vale_2.28.0_Linux_64-bit.tar.gz -C bin
+   export PATH=./bin:"$PATH"
+   ```
+
+   ```macos
+   brew install vale
+   ```
+
+   {{< /code >}}
+
+1. Create a `vale.ini` file in your home directory or in a working directory with the following contents:
+
+   ```bash
+   MinAlertLevel = suggestion
+   StylesPath = /FULL_PATH_TO_REPO/writers-toolkit/vale
+   [*.md]
+   BasedOnStyles = Google, Grafana
+   Google.Quotes = NO
+   Google.Units = NO
+   Google.WordList = NO
+   TokenIgnores = (<http[^\n]+>+?)
+   ```
+
+   Replace `FULL_PATH_TO_REPO` with the full path to the cloned Technical Documentation repository. For example, in Linux you could set StylesPath to `/home/username/git-repos/technical-documentation/linters/vale` and in macOS, you could set it to `/Users/username/git-repos/technical-documentation/linters/vale`. The path depends on where you cloned the git repository.
+
+1. Install the [Vale VS Code extension](https://marketplace.visualstudio.com/items?itemName=chrischinchilla.vale-vscode) in VS Code.
+
+   1. Start VS Code.
+   1. Press Ctrl+P, paste the following command, and press Enter.
```

Comment:
Wondering if we might also just add how  you can do this in the UI (i.e., Click Extensions, search for Vale VS Code, click Install."

Reply:
We could... I never use that route, so it never occurred to me to add it :-P

Reply:
Using the UI always occurs to me, lol. I hate keyboard commands. They're like a weird hidden language to me. I've made a suggestion below.